packages/uploadthing/src/internal/express.ts

```diff
@@ -34,16 +32,10 @@ export const createUploadthingExpressHandler = <TRouter extends FileRouter>(
       return;
     }
 
-    const { data } = bodyResult;
-
     const response = await requestHandler({
-      req: {
-        ...req,
-        // Explicitly passing headers since
-        // without that they are being lost for some reason
-        headers: req.headers,
-        json: () => new Promise((resolve) => resolve(data)),
-      },
+      req: Object.assign(req, {
+        json: () => Promise.resolve(bodyResult.data),
+      }),
```

Comment:
this seems to fix the header situation @Mr0Bread

Reply:
Cool, thank you